minmea.c

```diff
@@ -98,18 +98,20 @@ bool minmea_scan(const char *sentence, const char *format, ...)
 
     const char *field = sentence;
 #define next_field() \
-    do { \
-        /* Progress to the next field. */ \
-        while (minmea_isfield(*sentence)) \
-            sentence++; \
-        /* Make sure there is a field there. */ \
-        if (*sentence == ',') { \
-            sentence++; \
-            field = sentence; \
-        } else { \
-            field = NULL; \
-        } \
-    } while (0)
+    if(sentence) { \
```

Comment:
also this preferably shouldn't be inside the workflow pull request, it's an unrelated change